.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis
+on:
+  pull_request:
+
+jobs:
+  static_analysis:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '12.x'
+
+    - name: Set up Dart SDK
+      uses: dart-lang/setup-dart@v1
+      with:
+        dart-sdk: '2.17.6'
+
+    - uses: subosito/flutter-action@v1
```

Comment:
Could you try upgrading to `subosito/flutter-action@v2`? (see: https://github.com/subosito/flutter-action)